README.md

```diff
@@ -182,7 +182,8 @@ FLAGS:
 
 OPTIONS:
     -a, --accept <accept>                      Comma-separated list of accepted status codes for valid links
-    -b, --base-url <base-url>                  Base URL to check relative URLs
+    -b, --base <base>                          Base URL or website root directory to check relative URLs e.g.
```

Comment:
I suggest to to call it "internal URLs" or even "absolute internal URLs" instead of "relative URLs". From my understanding:
- relative internal URL: `path/to/here` or `../path/to/there`
    - no leading slash, where a base path/URL can be derived from the input document's path/URL directly (appending + optionally resolving dot path elements), hence no additional option is required.
- absolute internal URL: `/path/to/elsewhere`
    - leading slash, where a base path/URL is indeed required, as it cannot be derived relative from the input document's path/URL.
- external URL: `https://example.org/stairway/to/heaven`

Reply:
base URL of HTML has a definition:

https://html.spec.whatwg.org/multipage/urls-and-fetching.html#document-base-url

Whatever option name is used, it must comply with the specification.

Reply:
Thanks for the links. So it's "path-relative-URL string" and "path-absolute-URL string" that I usually name internal links, while indeed they are both "relative-URL strings". So the "relative" in the first case means "relative path" while in the second case it means "relative to any leading component of the full URL" 😅.

A separation is still required, as for "path-relative-URL strings", no base path/URL is required, while for "path-absolute-URL strings" it is. And I'm not fully convinced that "Base URL or website root directory to check path-relative-URL strings e.g." is great here 🤔. On the other hand, reading it again, not to bad, and correct. Not me to decide 🙂.